hack/make/.build-rpm/docker-core.spec

```diff
@@ -0,0 +1,104 @@
+Name: docker-core
+Version: %{_rpmversion}
+Release: 1%{?dist}
+Summary: The open-source application container engine
+
+License: ASL 2.0
+Source: docker-core.tar.gz
+
+URL: https://dockerproject.com
+Vendor: Docker
+Packager: Docker <support@docker.com>
+
+BuildRequires: systemd
+Requires: /bin/sh
+Requires: libc.so.6
+Requires: libdevmapper.so.1.02
+Requires: libpthread.so.0
+Requires: libsqlite3.so.0
+Requires: iptables
+
+Conflicts: docker
+Conflicts: docker-io
+Conflicts: lxc-docker
+Conflicts: lxc-docker-virtual-package
```

Comment:
true i think just docker & docker-io apply i just put in the others bc why
not lol? but can remove

On Fri, May 1, 2015 at 1:23 PM, Tianon Gravi <notifications@github.com>
wrote:

> In hack/make/.build-rpm/docker-core.spec
> <https://github.com/docker/docker/pull/12917#discussion_r29530625>:
>
> > +URL: https://dockerproject.com
> > +Vendor: Docker
> > +Packager: Docker <support@docker.com>
> > +
> > +BuildRequires: systemd
> > +Requires: /bin/sh
> > +Requires: libc.so.6
> > +Requires: libdevmapper.so.1.02
> > +Requires: libpthread.so.0
> > +Requires: libsqlite3.so.0
> > +Requires: iptables
> > +
> > +Conflicts: docker
> > +Conflicts: docker-io
> > +Conflicts: lxc-docker
> > +Conflicts: lxc-docker-virtual-package
>
> Are all of these packages that we actually need to conflict with? In the
> Debian version, we had to conflict against every previous "Docker" package,
> so here we probably need the same but I don't know if all of these have
> actually existed as RPMs before.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12917/files#r29530625>.
>